gravitee-apim-portal-webui/src/app/pages/api/api-subscribe/api-subscribe.component.ts

```diff
@@ -678,14 +679,14 @@ export class ApiSubscribeComponent implements OnInit {
   }
 
   private canSubscribe(appSubscriptions: Subscription[], plan: Plan) {
-    if (appSubscriptions && appSubscriptions.length > 0 && plan.security.toUpperCase() !== Plan.SecurityEnum.SUBSCRIPTION) {
+    if (appSubscriptions && appSubscriptions.length > 0 && plan.security?.toUpperCase() !== Plan.SecurityEnum.SUBSCRIPTION) {
```

Comment:
`Plan.SecurityEnum.SUBSCRIPTION` should not exist anymore. I think we have to find another condition for this test

Reply:
Done